src/components/templates/product-availability-modal/index.tsx

```diff
@@ -0,0 +1,71 @@
+import { SalesChannel } from "@medusajs/medusa"
+import React from "react"
+import Button from "../../fundamentals/button"
+import Modal from "../../molecules/modal"
+import LayeredModal, {
+  LayeredModalContext,
+} from "../../molecules/modal/layered-modal"
+import AvailableChannelsModalScreen from "./available-channels-modal-screen"
+
+type ProductAvailabilityModalProps = {
+  salesChannels: SalesChannel[]
+  storeSelectedSalesChannels: (salesChannels: SalesChannel[]) => void
+  handleClose: () => void
```

Comment:
suggestion: `onClose`? like we have `onSave` instead of `handleSave` just for consistency